webpack.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     publicPath: path.resolve("/"),
   },
   devServer: {
-    contentBase: path.resolve(__dirname, "src", "public"),
+    contentBase: path.resolve(__dirname, "public"),
```

Comment:
Wait so now public-facing HTML pages are located at `$PROJECT_ROOT/public` instead of `$PROJECT_ROOT/public`. This is intended right?

Reply:
So you mean from `$PROJECT_ROOT/src/public`? Yes, as this is the default common vue cli project structure so I think I might just move it out of src for better separation :D